packages/internal/src/build/babel/web.ts

```diff
@@ -111,7 +113,7 @@ export const getWebSideOverrides = (
     // Automatically import files in `./web/src/pages/*` in to
     // the `./web/src/Routes.[ts|jsx]` file.
     {
-      test: /Routes.(js|tsx)$/,
+      test: /Routes.(js|tsx|jsx)$/,
```

Comment:
Do we really support .jsx files?

Reply:
Yes we do - read this for context: https://github.com/vitejs/vite/discussions/3448

I have a plugin temporarily so that you don't need to switch to .jsx, but once Vite is stable, I think we should move everything to .jsx as per the thread.

Reply:
I knew I remembered something about a js vs jsx conversation! https://github.com/redwoodjs/redwood/issues/734

Reply:
So we do support .jsx, we just didn't want to make it the default 🙂  (That was a long time ago though, so people might have changed their name)

Reply:
> ...once Vite is stable, I think we should move everything to .jsx as per the thread

We've discussed this once before and a lot of the team was against this. So queue it up for discussion again when you're ready